### PR TITLE
Fix/prebuilt doc

### PR DESCRIPTION
**Pull request by saikatmitra91**

### CODE REVIEW

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 1]
```diff
@@ -3,13 +3,15 @@ title: 100ms Prebuilt
 nav: 4.3
 ---
 
-100ms Prebuilt is a feature-rich video conferencing/live streaming [web app](https://github.com/100mslive/100ms-web/) made using our prebuilt components. Prebuilt allows users to join sharable room links via desktop, meeting invites, or as an embeddable experience using an [iFrame](/docs/get-started/v2/get-started/prebuilt/prebuilt#share-or-embed-as-iframe), or as a native component in your apps.
+100ms Prebuilt link is a feature-rich video conferencing/live streaming [web app](https://github.com/100mslive/100ms-web/).
```
Comment (arjunattam) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:6]: intro is outdated ("web app"). is that fixed in the base branch?

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 2]
```diff
@@ -46,7 +48,7 @@ In this guide, we will create role specific prebuilt room links and share/embed
 <FlexContainer>
 <Callout icon="link" title="Prebuilt Link Structure">
 Prebuilt links are role specific room links with a format `https://<template-subdomain>.app.100ms.live/<room-code>` where,
-    - `template-subdomain`- is the subdomain that you’ve defined at the time of template creation on [100ms Dashboard](https://dashboard.100ms.live/dashboard). Copy `template-subdomain` from Template details page on 100ms Dashboard
+    - `template-subdomain`- is the subdomain that you’ve defined at the time of template creation on [100ms Dashboard](https://dashboard.100ms.live/dashboard). Copy `subdomain` from Template details page on 100ms Dashboard
```
Comment (arjunattam) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:51]:
```suggestion
    - `template-subdomain`- is the subdomain that you’ve defined at the time of template creation on [100ms Dashboard](https://dashboard.100ms.live/dashboard). Copy the subdomain (e.g. "videoconf-app") from Template details page on 100ms Dashboard
```

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 3]
```diff
@@ -128,7 +131,7 @@ import 'package:hms_room_kit/hms_room_kit.dart';
 // add pre-built component
 HMSPrebuilt(roomCode: "<room-code>");
 
-```
+````
```
Comment (arjunattam) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:134]: 4 backticks?

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 4]
```diff
@@ -75,26 +78,27 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
...
+
 </iframe>
 ```
 
 </StepsContainer>
 
 ## Embed as Component
 
-100ms Prebuilt can be embedded as a native component `HMSPrebuilt` by importing HMSRoomKit in your apps.
+100ms Prebuilt can be embedded as a native component `HMSPrebuilt` by importing `RoomKit` package in your app.
```
Comment (arjunattam) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:90]: this needs more context. we're introducing a major thing here but the copy is not doing justice to it. need to elaborate on which platforms this is available, a line on what customisation is possible?

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 5]
```diff
@@ -75,26 +78,27 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
...
 
 <FlexContainer>
 <Callout icon="link" title="HMSPrebuilt Component Basics">
+    - **RoomKit** -  Collection of embeddable 100ms UI components for your app.
+        - `HMSPrebuilt` is the first embeddable UI component of RoomKit package.
+            - Have a Component request? Drop a note here
```
Comment (arjunattam) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:96]: "Drop a note here" - where?

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 6]
```diff
@@ -75,26 +78,27 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
...
 
-    - **HMSPrebuilt -** high-order component from HMSRoom Kit that handles all audio/video functionality within a room
-        - has default colors and styles of 100ms, which can be overridden from the dashboard.
+    - **HMSPrebuilt** - high-order component from HMSRoom Kit that handles all audio/video functionality within a room.
```
Comment (arjunattam) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:98]: does this need to be another bullet point? how is it different from line 95?

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx (hunk 1 above)
Comment (saikatmitra91) [reply docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:6]: naah base branch content is on the left. 

let me comeback with better intro

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx (hunk 2 above)
Comment (saikatmitra91) [reply docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:51]: actually this is still not the best. 

dashboard helps you to copy the domain. 
need to fix this.

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx (hunk 5 above)
Comment (saikatmitra91) [reply docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:96]: wil be asking @akankshat100ms  to add it in base branch

**Pull request merged**